Kazim Hassan

E-Team opened the pull request.

E-Team committed:
Fingers Crossed!!

mindlink commented:
Hi Kazim,

Thank you for your submission, however it appears to us that you have missed the point of this exercise. Is this your final submission?

For instance, we don't see any implementation of any of the features detailed in the readme. As per the readme we won't consider any submissions that do not _at least_ include the essential features.

MindLink

E-Team committed:
Attempted all features apart from last additional feature. 


Obfuscate feature although implemented but not sure when to use it. An
example of result would have helped.

E-Team commented:
Really sorry about that. Not going to explain what happened or it will result in me getting fired before am even hired. 

Thanks,
Kazim

E-Team committed:
Test Cases Updated.

E-Team committed:
Test Cases Implemented.

E-Team commented:
Final Submission. Thanks
